### DISCUSSION TITLE
Extra Documentation for Anchors

### DISCUSSION THREAD

**Discussion by GiantRobots** [twilio-labs/paste#1240] [Ideas] [answered]
We have this documentation which is excellent around rel=noreferrer and noopener

![Screen Shot 2021-03-02 at 5 00 26 PM](https://user-images.githubusercontent.com/1903839/109731588-cb2dcb00-7b78-11eb-92a0-48ab48aef4fc.png)

I think we should enhance this with a few links to other references like to this MDN page for referrer policy https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Referrer-Policy . I think it may be worthwhile telling people for extra security adding headers or meta tags with the appropriate settings can set the referrer for all links ex.

```html
<meta name="referrer" content="noreferrer noopener">
```

**Comment by SiTaggart** [comment, marked as answer]
Hi @GiantRobots 

Would you consider adding this as a PR submission? Seems like a great idea to include references to me.

**Reply by gloriliale** [reply]
Hi @GiantRobots! Would you still be interested in submitting this as a PR?
